### DISCUSSION TITLE
Configure IP of Sonos Speaker

### DISCUSSION THREAD

**Discussion by CodingEmil** [music-assistant/support#2422] [Q&A] [answered]
Hello guys, I have my Sonos Speakers in VLAN1 and my Home Assistant in VLAN2. This means Music Assistant won't be able to scan for the speakers. 
Is there any option to configure the IPs of the Sonos speakers? To skip the scanning.

**Comment by OzGav** [comment, marked as answer]
Sorry we don’t supply support for complex network setups. https://music-assistant.io/faq/troubleshooting/
